user/using-debug-vm.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Debug VM
+layout: en
+permalink: /user/using-debug-vm/
+---
+
+When the build failure is only observable on Travis CI, and if you suspect that there are
+critical differences between your local development environment and Travis CI's build VM,
+you can restart builds in the debug mode to gain access to the actual VM or container
+(henceforth "debug VM").
+
+# Restarting a build to debug
+
+This feature is available for private repositories and those public repositories for which
+the feature is enabled.
+To have the feature enabled on a public repository, please email us at support@travis-ci.com
+indicating which ones.
+
+In addition, the push access to the repository is required.
```

Comment:
What about adding this requirement as a note/warning? Perhaps right before the `Starting a debug VM via API` section or at the beginning of this one.

I'd rephrase it to something similar to: It is required to have push access to the repository to start a debug build.